HelpSource/Classes/MainMenu.schelp

```diff
@@ -0,0 +1,100 @@
+TITLE:: MainMenu
+summary:: A manager for application-level menus.
+categories:: GUI
+related:: Classes/MenuAction, Classes/ToolBar, Classes/MainMenu
+
+DESCRIPTION::
+MainMenu is used to control items displayed in the application-level menu.
```

Comment:
It might be good to mention that this is the sclang menu, not the IDE one. This can be very confusing if you don't know it.

Reply:
I was also confused by this; added a note